lib/rake/build.rake

```diff
@@ -186,7 +186,20 @@ namespace :build do
     end
   end
 
+  desc 'Builds frontend'
+  timed_task_with_logging :frontend do
+    Dir.chdir(frontend_dir) do
+      ChatClient.log 'Installing <b>frontend</b> dependencies...'
+      RakeUtils.yarn_install
+
+      # Only build frontend packages that are relevant to `code-dot-org/apps`
+      ChatClient.log 'Building <b>frontend</b>...'
+      RakeUtils.system "yarn release:dryrun --filter @code-dot-org/dsco "
+    end
+  end
+
   tasks = []
+  tasks << :frontend if CDO.build_apps
```

Comment:
Is order important? e.g. does frontend need to be built before apps? If order matters, I'd prefer a brief comment (not that we have them between apps and dashboard lol)

Reply:
`frontend` does need to be built before `apps` since apps consumes the design system. I'll add in a comment.